.travis.yml

```diff
@@ -49,7 +50,7 @@ install:
   - |
    if [ "${SYMFONY_VERSION}" != "" ]; then
       composer config --unset platform.php
-      composer require \
+      travis_retry composer require \
```

Comment:
why is a `travis_retry` required here?

Reply:
I had a failing build because composer could not install these here.